app/uk/gov/hmrc/cbcr/controllers/ReportingEntityDataController.scala

```diff
@@ -133,6 +133,20 @@ class ReportingEntityDataController @Inject()(repo: ReportingEntityDataRepo, aut
       }
   }
 
+  def isOverlapping(tin: String, startDate: String, endDate: String) = auth.authCBCR { implicit request =>
+    repo
+      .queryTINDatesOverlapping(tin, EntityReportingPeriod(LocalDate.parse(startDate), LocalDate.parse(endDate)))
```

Comment:
If these LocalDate.parse fail, you're going to get the error below. Is it possible it could fail the parse?

Reply:
The dates are validated in the frontend before hand so didn't bother to validate again. And if they fail is reasonable to get the error.